.github/workflows/ci.yml

```diff
@@ -37,3 +37,26 @@ jobs:
       - run: yarn install --immutable
       - run: yarn run build
       - run: yarn run test
+
+  docker-publish:
+    if: (github.event_name == 'push' && github.ref == 'refs/heads/main')
+    needs: [ prettier, es-lint, tests-finish ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-qemu-action@v2.1.0
+        with:
+          platforms: arm64
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: docker/build-push-action@v3
+        env:
+          DOCKER_IMAGE_TAG: safeglobal/safe-gelato-relay-service:staging
```

Comment:
Why did you extract this variable? It's only used in one place. I  don't think we should set it in the `env`